dmutils/apiclient/data.py

```diff
@@ -412,6 +412,14 @@ def update_service_status(self, service_id, status, user):
                 },
             })
 
+    def find_frameworks(self):
+        return self._get("/frameworks")
+
+    find_frameworks_iter = make_iter_method('find_frameworks', 'frameworks', 'frameworks')
```

Comment:
Is the `/frameworks` endpoint paginated? I doubt we really need the `_iter` for it, but I guess it doesn't hurt as long as it works.